### ISSUE TITLE
Model Implementation Tools

### ISSUE THREAD

**Issue by paulhibbing** [wadpac/oss-dev-webinar-series-pb-field#2]
In the spirit of #1, I thought it would be useful to know what channels currently exist for implementing new methods (for counts or raw data) as they're published. A few I'm aware of are:

- [PhysicalActivity](https://cran.r-project.org/package=PhysicalActivity) for implementing the Choi wear/non-wear algorithm
- [TwoRegression](https://cran.r-project.org/package=TwoRegression) for two-regression algorithms
- [Sojourn](https://cran.r-project.org/package=Sojourn) and [Sojourn.Data](https://cran.r-project.org/package=Sojourn.Data) for Sojourn algorithms
- [GGIR](https://cran.r-project.org/package=GGIR) for sleep, wear/non-wear, and activity

As time goes on, outlets like this have the potential to make publication and uptake of new methods much cleaner. But we'll need an ongoing dialog around what the best dissemination approaches are, since we'll likely run into situations where it's difficult to balance strategy and courtesy when it comes to package scope/functionality versus package size. (How many neural networks should go in a dedicated package before it becomes more courteous to split up the package and allow partially-selective access to certain objects?)

**Comment by vincentvanhees** [comment]
I fully agree. Jairo and I listed a few raw data specific tools in our recent [paper](https://journals.humankinetics.com/view/journals/jmpb/2/3/article-p188.xml). Here a brief summary: 

- [GENEAclassify](https://cran.r-project.org/web/packages/GENEAclassify/index.html), I am not sure whether there is a GitHub repo for this.
- [Pampro](https://github.com/Thomite/pampro), for which one of its [forks](https://github.com/MRC-Epid/pampro/tree/master/pampro) has an open source license as of recently.
- [Biobank-Accelerometer-Analysis](https://github.com/activityMonitoring/biobankAccelerometerAnalysis)
- [OMgui](https://github.com/digitalinteraction/openmovement/wiki/AX3-GUI)
